doc/concepts/data_model/value_store.rst

```diff
@@ -714,32 +716,23 @@ Creating foreign keys
     For each foreign key, there must exist an index that includes all its fields.
 
 To create a foreign key in a space, specify the parent space and linked fields in the ``foreign_key`` parameter.
+Parent spaces can be referenced by name or by id. When linking to the same space, the space can be omitted.
 Fields can be referenced by name or by number:
 
 *   Field foreign keys: when setting up the space format.
 
-    ..  code-block:: tarantoolsession
-
-        tarantool> box.space.orders:format({
-                 > {name = 'id',   type = 'number'},
-                 > {name = 'customer_id', foreign_key = {space = 'customers', field = 'id'}}, -- or field = 1
-                 > {name = 'price_total',  type = 'number'},
-                 > })
+    ..  literalinclude:: /code_snippets/test/foreign_keys/field_foreign_key_test.lua
```

Comment:
Including the index without the parent space creation and formatting doesn't make this snippet copy-pastable.
On the other hand, including everything about the parent space doubles the snippet size, making the mechanism look heavy and complex.

Reply:
Generally, I agree. As an idea for the future, it would be convenient to have smth like the `Open full sample` button in the right corner to have the capability to open the example right on the GithHub and understand the full context.